agent/client.rb

```diff
@@ -20,6 +20,7 @@ def request_run_fastlane(bin, *params, env: {})
       end
     end
   end
+  @file && @file.close
```

Comment:
What is this line? I don't understand, can this be inside a method?

Reply:
This Client is just a playground for testing the agent outside fastlane.ci - not sure if we want to keep it but I thought it can come handy at times and decided to keep. 
That last line seems to close the file containing the streamed-back artifact, if it exists.